test/e2e/suite/trustpolicy/registry_scope.go

```diff
@@ -16,7 +16,7 @@ var _ = Describe("notation trust policy registryScope test", func() {
 			vhost.SetOption(AddTrustPolicyOption("empty_registry_scope_trustpolicy.json"))
 
 			// test localhost:5000/test-repo
-			OldNotation().Exec("sign", artifact.ReferenceWithDigest()).MatchKeyWords(SignSuccessfully)
+			notation.Exec("sign", artifact.ReferenceWithDigest()).MatchKeyWords(SignSuccessfully)
```

Comment:
Why did we originally ran test using old release?

Reply:
We are testing `empty registry scope` in this test case. So yes, using the `OldNotation` should also work. Will change it back.